src/unit-tests/loginAndLogout.test.js

```diff
@@ -0,0 +1,34 @@
+import { login } from "../js/api/auth/login.js";
+import { logout } from "../js/api/auth/logout.js";
+const mockLocalStorage = (() => {
```

Comment:
You extract this to a separate file and import here

Reply:
I am not sure what you mean by this, but readme and the unit test are fixed